src/metatensor/models/cli/export.py

```diff
@@ -27,9 +34,30 @@ def _add_export_model_parser(subparser: argparse._SubParsersAction) -> None:
     parser.add_argument(
         "-o",
         "--output",
-        dest="path",
+        dest="output",
         type=str,
         required=False,
         default="exported-model.pt",
         help="Filename of the exported model (default: %(default)s).",
     )
+
+
+def export_model(
+    model: torch.nn.Module, output: Union[Path, str] = "exported-model.pt"
+) -> None:
+    """Export a trained model to allow it to make predictions.
+
+    This includes predictions within molecular simulation engines. Exported models will
+    be saved with a ``.pt`` file ending. If ``path`` does not end with this file
+    extensions ``.pt`` will be added and a warning emitted.
+
+    :param model: model to be exported
+    :param output: path to save the exported model
+    """
+    path = str(check_suffix(filename=output, suffix=".pt"))
+
+    if is_exported(model):
+        torch.jit.save(model, path)
+    else:
+        exported_model = export(model)
+        exported_model.export(output)
```

Comment:
Is this function for saving a model? In that case, shouldn't it be named save? I see that one branch also exports, but wasn't the idea to separate export and save?

Reply:
Sorry, this is the main export function, got it now. Should we raise a warning if someone is trying to export an exported model?